src/main/io/gps.c

```diff
@@ -512,7 +597,7 @@ static void ubloxSendNAV5Message(uint8_t model)
     tx_buffer.payload.cfg_nav5.reserved0[0] = 0;
     tx_buffer.payload.cfg_nav5.reserved0[1] = 0;
     tx_buffer.payload.cfg_nav5.staticHoldMaxDist = 200;
-    tx_buffer.payload.cfg_nav5.utcStandard = 0;
+    tx_buffer.payload.cfg_nav5.utcStandard = ubloxUTCStandardConfig_int[gpsConfig()->gps_ublox_utc_standard];
```

Comment:
Same potential critical bug here

Reply:
no it works like this because the setting writes 0 - 5 into the config array. The int is just a mapping from 0-5 to the corresponting value for the ublox message. Can we assign the correct ints to the config values directly? If yes, we can remove the mapping array.